spec/factories/address.rb

```diff
@@ -49,9 +49,17 @@
         n
       end
 
-      sequence :country_iso do |n|
-        "country_iso_#{n}"
-      end
+      uprn { nil }
+      premises { nil }
+      street_address { nil }
+      locality { nil }
+      city { nil }
+      postcode { nil }
+      country_iso { nil }
+    end
+
+    trait :site_uses_address do
+      address_type { 3 }
```

Comment:
NIT 
```suggestion
      address_type { :site }
```
This will work thanks to `enum` and is more readable IMO :)

Reply:
Think that was how I found it, but as I'm making other changes this makes perfect sense.